units/systemd-networkd-wait-online.service.in

```diff
@@ -14,7 +14,7 @@ ConditionCapability=CAP_NET_ADMIN
 DefaultDependencies=no
 Conflicts=shutdown.target
 BindsTo=systemd-networkd.service
-After=systemd-networkd.service
+After=systemd-networkd.service systemd-resolved.service
```

Comment:
Please drop the change. `--dns` is not enabled in the command line, hence not necessary.